script/openqa-bootstrap

```diff
@@ -108,7 +108,7 @@ if command -v $setup; then
 else
     curl -s https://raw.githubusercontent.com/os-autoinst/openQA/master/script/configure-web-proxy | bash -ex -s -- "$proxy_args"
 fi
-sed -i -e 's/#*.*method.*=.*$/method = Fake/' /etc/openqa/openqa.ini
+sed -i -e 's/#*.*method.*=.*$/method = Fake/' -e 's/#git_auto_clone =.*$/git_auto_clone = yes' /etc/openqa/openqa.ini
```

Comment:
why is this not already enabled by default in openQA?

Reply:
Because it is new. And this way it was easier to merge the feature it without fearing to break anything right away.

Reply:
To avoid breaking existing production instances in case the feature misbehaves. The feature might also have security implications that could be relevant for instances used by the public. And because there's no cleanup yet.

(The last point makes enabling this actually generally questionable but I hope it is ok for new instances setup by the bootstrap script.)

Reply:
ok. I created https://progress.opensuse.org/issues/162077 to follow up with that